app/lib/utils.js

```diff
@@ -7,3 +9,7 @@ export const formatAttachmentUrl = (attachmentUrl, userId, token, server) => {
 	}
 	return encodeURI(`${ server }${ attachmentUrl }?rc_uid=${ userId }&rc_token=${ token }`);
 };
+
+export const isServerVersionLowerThan = (currentServerVersion, oldServerVersion) => currentServerVersion && lt(coerce(currentServerVersion), oldServerVersion);
+
+export const isServerVersionGreaterThan = (currentServerVersion, oldServerVersion) => currentServerVersion && gte(coerce(currentServerVersion), oldServerVersion);
```

Comment:
I have concerns about this naming still.
`server greater than` is actually `greater than or equal to`.
That might cause some issues in the future.
Do you agree?

What if we do something like this?
```js
export const methods = {
	lowerThan: lt,
	lowerThanOrEqualTo: lte,
	greaterThan: gt,
	greaterThanOrEqualTo: gte
};

export const compareServerVersion = (currentServerVersion, versionToCompare, func) => currentServerVersion && func(coerce(currentServerVersion), versionToCompare);
```

Usage would be like this:
```js
console.log('1.0.0 lower than 1.1.0', compareServerVersion('1.0.0', '1.1.0', methods.lowerThan));
console.log('1.0.0 lower than or equal to 1.1.0', compareServerVersion('1.0.0', '1.1.0', methods.lowerThanOrEqualTo));
console.log('1.0.0 greater than 1.1.0', compareServerVersion('1.0.0', '1.1.0', methods.greaterThan));
console.log('1.0.0 greater than or equal to 1.1.0', compareServerVersion('1.0.0', '1.1.0', methods.greaterThanOrEqualTo));
```

WDYT? @gerzonc